static/src/javascripts/projects/common/modules/navigation/new-header.js

```diff
@@ -262,6 +262,16 @@ const initiateUserAccountDropdown = (): void => {
                     toggleDropdown(userAccountDropdownEls)
                 );
             }
+
+            const { menu } = userAccountDropdownEls;
+
+            if (menu) {
+                menu.addEventListener('keyup', (event: KeyboardEvent): void => {
+                    if (event.key === 'Escape') {
+                        toggleDropdown(userAccountDropdownEls);
+                    }
+                });
```

Comment:
should we remove the event listener after closing?

Reply:
`initiateUserAccountDropdown` is only called once as part of those modules initialisation so we're not adding this listener every time the user opens/closes them menu.